programs/server/Server.cpp

```diff
@@ -257,6 +257,7 @@ namespace ServerSetting
     extern const ServerSettingsUInt64 max_partition_size_to_drop;
     extern const ServerSettingsUInt64 max_part_num_to_warn;
     extern const ServerSettingsUInt64 max_pending_mutations_to_warn;
+    extern const ServerSettingsUInt64 max_threshold_to_warn_for_pending_mutations;
     extern const ServerSettingsUInt64 max_parts_cleaning_thread_pool_size;
```

Comment:
Maybe better smth like `max_pending_mutations_execution_time_to_warn`?

Reply:
Thank you. Yeah, that sounds better to me and is more uniform with the other setting above - changed.